src/pinns_pde_solve.jl

```diff
@@ -450,11 +466,13 @@ function DiffEqBase.discretize(pde_system::PDESystem, discretization::PhysicsInf
 
     pde_loss_function = get_loss_function(eval(expr_pde_loss_function),
```

Comment:
for safety, you probably want to use RuntimeGeneratedFunctions.jl here for the eval.